engine/lib/output.php

```diff
@@ -153,6 +153,12 @@ function elgg_format_attributes(array $attrs = array()) {
 
 		unset($attrs['js']);
 	}
+	
+	if (isset($attrs['data-colorbox-opts'])) {
+		if (!@json_decode($attrs['data-colorbox-opts'])) {
```

Comment:
that was my first plan, but it is only doable "onclick" and there is no notice of debug level, so you can only limit it to admin only. We have no good debug reporting capabilities client side.